git/cmd.py

```diff
@@ -13,6 +13,8 @@
 import errno
 import mmap
 
+from collections import OrderedDict
```

Comment:
Of course: You will find these announcements in the [changelog](https://github.com/gitpython-developers/GitPython/blob/master/doc/source/changes.rst#200---features).
The cause of the move was some minor detail (which I forgot), so I believe after fixing this issue, 2.0 has a good change to run in py2.6 anyway. It's worth a try, considering how trivial the odict fix seems to be.